src/css/typography/_headings.scss

```diff
@@ -4,21 +4,49 @@ h1, h2, h3, h4, h5, h6 {
   text-align: left;
 }
 
+$h1-margin-top: 50px;
+$h2-margin-top: 45px;
+$h3-margin-top: 41px;
+$h4-margin-top: 39px;
+$h5-margin-top: 35px;
+$h6-margin-top: 30px;
+
 h1 {
-  @include block-formatting($size: 40px, $line-height: 46px, $weight: 700, $margin-top: 50px);
+  @include block-formatting($size: 40px, $line-height: 46px, $weight: 700, $margin-top: $h1-margin-top);
 }
 h2 {
-  @include block-formatting($size: 35px, $line-height: 43px, $weight: 600, $margin-top: 45px);
+  @include block-formatting($size: 35px, $line-height: 43px, $weight: 600, $margin-top: $h2-margin-top);
 }
 h3 {
-  @include block-formatting($size: 29px, $line-height: 35px, $weight: 600, $margin-top: 41px);
+  @include block-formatting($size: 29px, $line-height: 35px, $weight: 600, $margin-top: $h3-margin-top);
 }
 h4 {
-  @include block-formatting($size: 24px, $line-height: 29px, $weight: 600, $margin-top: 39px);
+  @include block-formatting($size: 24px, $line-height: 29px, $weight: 600, $margin-top: $h4-margin-top);
 }
 h5 {
-  @include block-formatting($size: 21px, $line-height: 28px, $weight: 700, $margin-top: 35px);
+  @include block-formatting($size: 21px, $line-height: 28px, $weight: 700, $margin-top: $h5-margin-top);
 }
 h6 {
-  @include block-formatting($size: 18px, $line-height: 22px, $weight: 700);
+  @include block-formatting($size: 18px, $line-height: 22px, $weight: 700, $margin-top: $h6-margin-top);
+}
+
+&.il-wp-block-editor-content {
```

Comment:
I don't think you need the & here

Reply:
I added both options, so you can do
`<div class="il-formatted il-wp-block-editor-content">...</div>` 
or
`<div class="il-formatted"><div class="il-wp-block-editor-content">...</div></div>`